spec/models/service/retirement_management_spec.rb

```diff
@@ -96,15 +99,13 @@
   end
 
   it "with one src_id" do
-    User.current_user = user
-    expect(ServiceRetireRequest).to receive(:make_request).with(nil, {:src_ids => ['yabadabadoo'], :__request_type__ => "service_retire"}, User.current_user, true)
-    @service.class.to_s.demodulize.constantize.make_retire_request('yabadabadoo', User.current_user)
+    expect(ServiceRetireRequest).to receive(:make_request).with(nil, {:src_ids => [service3.id], :__request_type__ => "service_retire"}, user, true)
```

Comment:
ASIDE: (only if this is a quick fix)

these tests are making us not call `make_request`. Makes me a little nervous to hide this code.

But I guess this is what the existing code does.
Just curious if it is quick to do this another way.